src/main/java/io/epirus/console/project/NewProjectCommand.java

```diff
@@ -127,16 +118,21 @@ public void run() {
                     case NONE:
                     case HELLOWORLD:
                         new OpenApiGeneratorService(
-                                        projectName,
-                                        packageName,
-                                        outputDir,
-                                        abis,
-                                        bins,
-                                        addressLength,
-                                        contextPath != null
-                                                ? StringUtils.removeEnd(contextPath, "/")
-                                                : projectName,
-                                        true)
+                                        new OpenApiGeneratorServiceConfiguration(
+                                                projectName,
+                                                packageName,
+                                                outputDir,
+                                                Collections.emptyList(),
+                                                Collections.emptyList(),
+                                                addressLength,
+                                                contextPath != null
+                                                        ? StringUtils.removeEnd(contextPath, "/")
+                                                        : projectName,
+                                                true,
```

Comment:
This smells a bit weird... two places in the code where the values are always `true` ? 🤔

Reply:
This will change after having the `web3j-openapi-gradle-plugin` working

Reply:
Won't merge this until we release a new version of web3j-openapi